### PR TITLE
Feature/71

### PR DESCRIPTION
**Pull request by Husterknupp**
@Bantolomeus see this one for the finished feature. Please answer my open question in #80 so we can decide how to continue.

Notes on AngularJs:
From the docs: 
>dynamic views in web-applications

(read: update values in html with very little manual boilerplate javascript code (https://angularjs.org/))

* your starting point to start reading from should be `index.html` -> `<nav-bar></nav-bar> <ui-view></ui-view>` where 
* nav-bar is a placeholder which is filled with `nav-bar.component.{html,js}`
* ui-view is a placeholder which is filled with `{overall-progress,todays-progress}.component.{html,js}`
* todays-progress in turn contains a component `progress-input.component.{html,js} (see AngularJs components: https://angularjs.org/#videoModal)
* `{{ book.book.name }}` in html files will tell AngularJs that there's a javascript variable to evaluate before the browser can render the html

If you wanna checkout how it looks, do as described in the README: 
* start the backend
* `npm install`
* `npm start`
* open `localhost:4002/` in your browser
* 
image

### CODE REVIEW

File: Api.md [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
-Payload: 
-
 ```
-{
+curl -X POST \
+  http://localhost:8080/challenge \
+  -H 'Content-Type: application/json' \
+  -d '{
     "pagesPerDay": 15,
     "pagesAheadOfPlan": 0,
```
Comment (Bantolomeus) [inline Api.md]:
```suggestion

```

File: Api.md [hunk 2]
```diff
@@ -1,23 +1,15 @@
...
     "startPagesAheadOfPlan": 0,
```
Comment (Bantolomeus) [inline Api.md]:
```suggestion
```

File: Api.md [hunk 3]
```diff
@@ -1,23 +1,15 @@
...
     "pagesSinceStart": 0,
```
Comment (Bantolomeus) [inline Api.md]:
```suggestion
```

File: Api.md [hunk 4]
```diff
@@ -1,23 +1,15 @@
...
     "pagesEverRead": 0,
```
Comment (Bantolomeus) [inline Api.md]:
```suggestion
```

**Comment by Bantolomeus** [comment]
As told in #80 I will remove the current pages left endpoint.

**Comment by Bantolomeus** [comment]
The current stand looks nice. I have a few suggestions for improvements but they should be addressed in there own issue.

**Comment by Husterknupp** [comment]
cool. Happy that this is useful for you. Happy to read your suggestions in their own issues.

**Comment by Husterknupp** [comment]
@Bantolomeus I understood your comment so that I am free to merge into master. Which I just did. 🙏  Thanks for the review 👁

**Comment by Bantolomeus** [comment]
That was right, I merged it myself

**Pull request merged**